examples/scaleway/README.md

```diff
@@ -0,0 +1,27 @@
+## Installation
+
+Add your credentials to launch the project. At least the following environement variables are required: `SCW_ACCESS_KEY,SCW_ACCESS_KEY,SCW_DEFAULT_ORGANIZATION_ID,SCW_DEFAULT_PROJECT_ID,SCW_DEFAULT_PROJECT_ID`.
+
+Configure the stack by modifying `inputs.tfvars` (e.g: cluster\_name) and launch the terraform apply with:
+
+```bash
+terraform init
+terraform apply -var-file inputs.tfvars 
+```
+
+## Usage
+Get the kubeconfig file and the domain name with the following commands:
```

Comment:
In my opinion, it's not best practice to use this Kubeconfig file as a way to authenticate to the cluster, since these are the same values used for setting the providers in the Terraform configuration. Maybe there is a way to recover new secret and ephemeral certificates using the Scaleway CLI utility, if it exists (see the SKS example).